rest/route/github.go

```diff
@@ -249,6 +249,16 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 				}
 			}
 		}
+
+	case *github.MetaEvent:
+		if event.Action == "deleted" {
+			hookID := event.GetHookID()
+			if hookID != 0 {
```

Comment:
Should this error if there's no hookID or is this potentially expected?

Reply:
It's unexpected. Fixed.